docs/DifferentiableProgramming.md

```diff
@@ -108,7 +108,7 @@ Backticks were added manually.
     *   [Not support differentiable programming](#not-support-differentiable-programming)
     *   [Use another language or framework for differentiable programming](#use-another-language-or-framework-for-differentiable-programming)
     *   [Other approaches to differentiable programming](#other-approaches-to-differentiable-programming)
-*   [Acknowledgements](#acknowledgements)
+*   [Acknowledgments](#acknowledgments)
```

Comment:
This is the British spelling, not a typo.

Reply:
Thanks, I didn't catch that. I fixed it.